manticore/models/query.py

```diff
@@ -25,6 +27,9 @@ def match(self, *args, **kwargs):
     def options(self, **kwargs):
         """ Adds OPTIONS clause to search query."""
         qs: SearchQuerySet = self._clone()
+        for key, value in kwargs.items():
+            if key == 'field_weights' and isinstance(value, dict):
```

Comment:
I suggest to add `field_weights` named arg to `options` signature and handle it separately